.github/workflows/continuous-integration.yml

```diff
@@ -121,15 +165,38 @@ jobs:
     runs-on: ubuntu-18.04
 
     steps:
-    - name: Install opensim-core
-      run: |
-        wget https://prdownloads.sourceforge.net/myosin/opensim-core/opensim-core-latest_linux_Release.zip
-        # -q: quiet
-        unzip -q opensim-core-latest_linux_Release.zip -d ~
 
     - name: Checkout opensim-gui
       uses: actions/checkout@v1
 
+    
+    - name: Checkout opensim-core master
+      uses: actions/checkout@v2
+      with:
+         name: opensim-org/opensim-core
+         repository: opensim-org/opensim-core
+         path: 'opensim-core'
+            
+    - name: Install packages
+      run: sudo apt-get install --yes liblapack-dev freeglut3-dev libxi-dev libxmu-dev doxygen python3 python3-dev python3-numpy python3-setuptools swig
+
+    - name: Build opensim-core-dependencies
+      run: |
+        mkdir build_deps
+        cd build_deps
+        cmake ../opensim-core/dependencies -DSUPERBUILD_ezc3d:BOOL=on -DCMAKE_INSTALL_PREFIX=~/opensim_dependencies_install
+        cmake . -LAH
+        cmake --build . --config Release
```

Comment:
I'm not sure `--config` has meaning for generators other than MSVC and Xcode. It's more typical to put `-DCMAKE_BUILD_TYPE=Release` when configuring.

Reply:
Left alone since it works as is and these experiments are expensive.